README.md

```diff
@@ -517,6 +518,66 @@ The SAP BTP service operator project maintainers will respond to the best of the
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
+## Formats of Secret Objects
+
+### Key- Value Pairs (Default)
+The binding object includes credentials returned from the broker and service instance info presented as key-value pairs.
+```bash
+#Credentials
+uri: https://my-service.authentication.eu10.hana.ondemand.com
+username: admin
+password: ********
+
+#Service instance info
+instance_guid: <instance_guid> // The service instance ID
+instance_name: my-service-btp-name // Taken from the service instance external_name field if set. Otherwise from metadata.name
+plan: sample-plan // The service plan name                
+type: sample-service  // The service offering name
+```
+
+### Credentials as JSON Object
+To show credentials returned from the broker as a JSON object, add 'secretKey: your-secret-value' to the binding spec. 
+
+your-secret-vaue will be a key for a value in JSON format that contains credentials while service instance info remains presented in key-value pairs.
+```bash
+#Credentials
+your-secret-value:
```

Comment:
it is a value/name of a key, not a secret 
use 'my-key' for example...

Reply:
ok changing